assets/admin/students/student-modal/index.test.js

```diff
@@ -9,6 +9,12 @@ import {
 	cleanup,
 } from '@testing-library/react';
 import nock from 'nock';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
```

Comment:
Suggestion, replace `useSelect` to  `useEntityRecords` 
`const { records, isResolving } = useEntityRecords( 'postType', 'page' );`